webapp/app/routes/room/room.styles.js

```diff
@@ -67,13 +78,30 @@ export const Container = styled.div`
 
 export const MenuDrawerInner = styled.div`
   width: ${DRAWER_WIDTH}px;
+  display:flex;
+  flex-direction: column;
+  height: 100%;
 `;
 
 export const MenuDrawerHeader = styled.div`
+  
   display: flex;
   align-items: center;
   justify-content: flex-end;
   padding: 0 ${theme.spacing.unit}px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.12);
+`;
+
+export const MenuDrawerContent = styled.div`
+  flex: 1;
+  overflow-y: auto;
+`;
+
+export const MenuDrawerFooter = styled.div`
+  height: 61px;
```

Comment:
można by to teraz dodać do zmiennej jako, że ta wysokość występuje w dwóch różnych miejscach